graylog2-web-interface/src/components/common/Select.jsx

```diff
@@ -289,7 +293,7 @@ class Select extends React.Component<Props, State> {
     };
   }
 
-  componentWillReceiveProps = (nextProps: Props) => {
+  componentWillReceivePropsProps = (nextProps: Props) => {
```

Comment:
I was just looking around and this didn't look right :)

Reply:
Hah, good catch! :)

Reply:
✅